blue-compile/src/main/scala/gnieh/blue/compile/SystemCompiler.scala

```diff
@@ -67,6 +67,10 @@ abstract class SystemCompiler(system: ActorSystem, config: Config, texmfcnf: Fil
     exec(s"bibtex main.aux", configuration.buildDir(paperId))
   }
 
+  def makeindex(paperId: String, settings: CompilerSettings)(implicit timeout: Timeout): Try[Boolean] = {
+    exec(s"makeindex build/main.idx", configuration.buildDir(paperId))
```

Comment:
Are you sure prefix with the `build` directory is necessary? The second argument gives the execution direction, which is already `build`